content/en/docs/reference/kubectl/docker-cli-to-kubectl.md

```diff
@@ -56,8 +62,7 @@ service "nginx-http" exposed
 
 By using kubectl, you can create a [Deployment](/docs/concepts/workloads/controllers/deployment/) to ensure that N pods are running nginx, where N is the number of replicas stated in the spec and defaults to 1. You can also create a [service](/docs/concepts/services-networking/service/) with a selector that matches the pod labels. For more information, see [Use a Service to Access an Application in a Cluster](/docs/tasks/access-application-cluster/service-access-application-cluster).
 
-By default images run in the background, similar to `docker run -d ...`. To run things in the foreground, use:
-
+By default images run in the background, similar to `docker run -d ...`. To run things in the foreground, use [`kubectl run`](/docs/reference/generated/kubectl/kubectl-commands/#run) to create pod:
```

Comment:
```suggestion
When you create a Deployment, the containers run in the background. This is roughly similar to `docker run -d ...` with extra features.
To wait for the container to exit, use [`kubectl run`](/docs/reference/generated/kubectl/kubectl-commands/#run) to create a Pod and attach to its container:
```

and also (_I think_) specify `--attach` for `kubectl run`.

Reply:
> and also (_I think_) specify `--attach` for `kubectl run`.

[It](https://github.com/kubernetes/website/pull/21707/commits/cf3becadff9eecc03e995a3f0ccfead0b2487eac#diff-16decda489c4f33805ead292d4af10ceR70) has been specially introduced.